scripts/management/MgmtDB.py

```diff
@@ -72,17 +73,19 @@ def close_conn(self):
         if self._conn is not None:
             self._conn.close()
 
-    def get_submitted_tasks(self):
+    def get_submitted_tasks(self, allowed_tasks=tuple(const.ProcessType)):
         """Gets all in progress tasks_to_run i.e. (running or queued)"""
+        allowed_tasks = [str(task.value) for task in allowed_tasks]
         with connect_db_ctx(self._db_file) as cur:
             result = cur.execute(
                 "SELECT run_name, proc_type, status, job_id, retries "
-                "FROM state WHERE status IN (2, 3)"
+                "FROM state WHERE status IN (2, 3) AND proc_type IN (?{})".format(",?"*(len(allowed_tasks)-1)),
```

Comment:
Do the join to make the table easier to read. Silly SQLite not having built in enums